features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NORDIC_CORDIO/TARGET_NRF5x/NRFCordioHCIDriver.cpp

```diff
@@ -316,19 +317,13 @@ void NRFCordioHCIDriver::start_reset_sequence()
 
 bool NRFCordioHCIDriver::get_random_static_address(ble::address_t& address)
 {
-    /* Load address from nRF configuration. */
-    uint64_t devAddr = (((uint64_t)NRF_FICR->DEVICEADDR[0]) <<  0) |
-                       (((uint64_t)NRF_FICR->DEVICEADDR[1]) << 32);
+    PalCfgLoadData(PAL_CFG_ID_BD_ADDR, address.data(), sizeof(bdAddr_t));
```

Comment:
Looking at sources, this bit of code loads the public address, not the random static one.

Reply:
get_random_static_address during reset

Reply:
So it loads the public address then not the random one ?